### DISCUSSION TITLE
after first update firmware get noise

### DISCUSSION THREAD

**Discussion by Mr-Rog** [Jyers/Marlin#228] [Q&A] [answered]
Hello
after i update first time, I get this nois, im sure im take a wrong setting
https://www.dropbox.com/s/eezp98tkbebet1a/ender3.mp4?dl=0
hope you can helping me

**Comment by Mr-Rog** [comment]
Hello 
I will reduce the speed, is this possible to insert fix to the firmware
![image](https://user-images.githubusercontent.com/60228815/110609525-97146480-818d-11eb-8dba-5566c508bfe7.png)
thx for a short feedback

**Comment by Schnabulation** [comment, marked as answer]
Yo Roger, I was just checking your video: As it seems you have a modified extruder (Bondtech, if I recall correctly) which needs different extrusion speeds. This is something you need to adjust in your personal firmware configuration (google for "calibrate e-steps") and not an issue with the firmware itself, as it is configured for the stock Ender 3 V2 extruder.

**Reply by Mr-Rog** [reply]
yes Steve  I calibrate the e-steps, there are fine, M92 E461.11, but I dont found the Gcode for Retractions Distance, but you have right its not the original extruder, maybe some one has me a hint for the Gcode but yes its not a Firmware issue

**Reply by Schnabulation** [reply]
...you set retraction distance in your slicer, not the firmware. :-)

**Comment by Mr-Rog** [comment]
yes, the question was, or thinking, it is possible to set in the firmware, or as you already mentioned only in my slicer
but i don't think so in that case, or you would have told me this
thanks for watching i will solve this the slicer way
cheers

**Reply by Schnabulation** [reply]
Well, the normal way is to set the retraction distance in your slicer - simply because different materials require different retraction settings. It wouldn't make sense to set this in the firmware.

**However:** There is a Marlin feature called "Firmware Retract". You can learn more about it here: https://marlinfw.org/docs/features/fwretract.html

**Comment by Mr-Rog** [comment]
ok that makes sense of course, if every filamment is different, since I rely entirely on your assessment and experience and will do exactly as you recommended, thank you for the help great cinema

**Reply by Schnabulation** [reply]
No worries man, everyone starts somewhere. ;-)

"Great cinema" - made me laugh... :-D
